apps/heft/src/utilities/TaskPackageResolver.ts

```diff
@@ -91,23 +114,27 @@ export class TaskPackageResolver {
     };
   }
 
-  private static _tryResolveTaskPackage(
+  private _tryResolveTaskPackage(
     taskPackageName: string,
-    projectFolder: string,
+    tsconfigBaseWithTypescriptDependencyFolder: string,
```

Comment:
Hmm, is this the best arg name? `packageFolder` made more sense before imo

Reply:
Ah I changed the wrong one.